src/main/java/org/alfresco/repo/search/impl/solr/SolrSQLJSONResultSet.java

```diff
@@ -83,7 +83,14 @@ public SolrSQLJSONResultSet(JSONObject json, BasicSearchParameters searchParamet
                 {
                     throw new RuntimeException("Unable to execute the query, this API requires InsightEngine.");
                 }
-                throw new RuntimeException("Unable to execute the query, error caused by: " + error);
+                if(searchParameters.getExtraParameters().containsKey("format"))
+                {
+                    

```

Comment:
I think you should either add a comment here or check the negative case and leave the else block.

Reply:
Removed redundant if statement